packages/flutter/lib/src/gestures/scale.dart

```diff
@@ -47,13 +48,15 @@ class ScaleStartDetails {
 class ScaleUpdateDetails {
   /// Creates details for [GestureScaleUpdateCallback].
   ///
-  /// The [focalPoint] and [scale] arguments must not be null. The [scale]
+  /// The [focalPoint], [scale] and [rotation] arguments must not be null. The [scale]
```

Comment:
nit: missing comma after `[scale]`

Reply:
Are you sure? That doesn't sound right. Like, we have an and right there connecting to [rotation]